maro/rl/utils/torch_utils.py

```diff
@@ -31,7 +31,7 @@ def match_shape(tensor: Union[torch.Tensor, np.ndarray], shape: tuple) -> bool:
         return True
 
 
-def ndarray_to_tensor(array: np.ndarray, device: torch.device) -> torch.Tensor:
+def ndarray_to_tensor(array: np.ndarray, device: torch.device = None) -> torch.Tensor:
```

Comment:
Is it okay to call `.to(device)` is `device` is None?

Reply:
Yes, .to(None) is equivalent to a no-op